boards.txt

```diff
@@ -252,6 +252,37 @@ STCT_nRF52_minidev.menu.softdevice.s132.build.extra_flags=-DNRF52 -DS132 -DNRF51
 STCT_nRF52_minidev.menu.softdevice.s132.build.ldscript=armgcc_s132_nrf52832_xxaa.ld
 
 
+Arduino_Primo_Core.name=Arduino Primo Core
+
+Arduino_Primo_Core.upload.tool=sandeepmistry:openocd
+Arduino_Primo_Core.upload.target=nrf52
+Arduino_Primo_Core.upload.maximum_size=524288
+
+Arduino_Primo_Core.bootloader.tool=sandeepmistry:openocd
+
```

Comment:
Does the Primo Core have a built-in programmer? (CMSIS DAP, JLINK, ST-LINK, etc?)

if so, a line should be added:
```
Arduino_Primo_Core.upload.protocol=<blah>
```

Reply:
The Primo Core needs an external programmer